variables.tf

```diff
@@ -88,4 +107,21 @@ variable "postgres_service" {
   })
   description = "Postgres data-object contains service_name, port, database_name, username and password"
 }
+variable "postgres_vault_secret" {
```

Comment:
Update readme section Inputs

Reply:
resolved in https://github.com/fredrikhgrelland/terraform-nomad-hive/pull/53/commits/48107b09b5abb90d42a3c727b8572e543f3aa0da